blog/models.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class User(AbstractUser):
+    class Meta:
+        verbose_name = "user"
+        verbose_name_plural = "users"
+
+    def __str__(self):
+        return f"{self.username}"
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(User,
+                              on_delete=models.CASCADE,
+                              related_name="posts")
```

Comment:
You might want to add a `verbose_name` and `verbose_name_plural` to the `Post` model to provide a human-readable name for the model. This is useful for the Django admin interface.